web/controllers/auth_controller.ex

```diff
@@ -32,6 +34,26 @@ defmodule Booky.AuthController do
 
     # Check if user belongs to the approved org
     if belong_to_github_org(client) do
+      # Check if user has already been added to the local database
+      unless Repo.get_by(User, username: user.username) do
+        # First login, insert into the db
+        changeset = User.changeset(%User{}, %{username: user.username, name: user.name})
+        case Repo.insert(changeset) do
+          {:ok, _} ->
+            conn
+            |> put_flash(:info, "Welcome to Booky!")
```

Comment:
This and the 2 pipelines below only differ in the flash messages, but I can't figure out how to abstract it out.

The flash messages stop working if I try.